pyfar/dsp/dsp.py

```diff
@@ -1528,7 +1537,7 @@ def deconvolve(system_output, system_input, fft_length=None, **kwargs):
     # multiply system_output signal with regularized inversed system_input
     # signal to get the system response
     system_response = (system_output *
-                       regularized_spectrum_inversion(system_input, **kwargs))
+                       regularized_spectrum_inversion(system_input, freq_range, **kwargs))
```

Comment:
1. Not flake8 conform and will not pass on CircleCI
2. Please consider saving the inverse spectrum in a separate variable for improved readability